src/openseadragon.js

```diff
@@ -1911,14 +1911,15 @@ function OpenSeadragon( options ){
 
 
         /**
-         * Sets the specified element's pointer-events style attribute to 'none'.
+         * Sets the specified element's pointer-events style attribute to the passed value.
          * @function
          * @param {Element|String} element
+         * @param {String} value
          */
-        setElementPointerEventsNone: function( element ) {
+        setElementPointerEvents: function( element, value ) {
```

Comment:
I imagine this isn't used a lot outside of OSD, but technically it is not private, so maybe it's worth keeping a `setElementPointerEventsNone` that calls this new one, just so we don't risk breaking someone who's using it.

Reply:
Oh yeah, thanks! I had it that way and then, thinking I could cut down on code bloat, I removed it :-/  Will fix!

Reply:
@iangilman Fixed!